ocaml/idl/datamodel_host.ml

```diff
@@ -2640,6 +2640,21 @@ let get_ntp_synchronized =
       )
     ~allowed_roles:_R_READ_ONLY ()
 
+let set_server_localtime =
+  call ~name:"set_server_localtime" ~lifecycle:[]
+    ~doc:
+      "Set the host's system clock in its local timezone when NTP is disabled."
```

Comment:
Using localtime is confusing because it's used for a very specific thing in UNIX: time + timezone, but this call is _not_setting the timezone. Please use something that doesn't use the term localtime, for example:
```suggestion
  call ~name:"set_time" ~lifecycle:[]
      ~doc:
      "Set the host's system clock, errors out when NTP is enabled."
```